store/promoted-offers.ts

```diff
@@ -10,6 +12,24 @@ export default interface PromotedOffersState {
   headImage: any[]
 }
 
+const getImages = async (url): Promise<any> => {
+  process.env.NODE_TLS_REJECT_UNAUTHORIZED = '0'
```

Comment:
same. do we really need this here in code? maybe we can conditionally add it via the configurations we saw today. what error did u get that you have to add this?

Reply:
this was the self-signed certificate error. would that not be a problem in prod?

Reply:
no it would not. can you try setting this environment variable in some configuration somewhere else? before you do. try to remove it and see if its still an issue.  bottom line is that we might need it in development mode, but not on production mode. so it can't be in the source code.

Reply:
hm seems to work fine now when I take it out we should be good

Reply:
just kidding it's still causing the error. is putting it in the .env file okay?